tracer/src/Datadog.Trace/AppSec/Transports/SecurityTransport.Core.cs

```diff
@@ -0,0 +1,154 @@
+// <copyright file="SecurityTransport.Core.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.Headers;
+using Datadog.Trace.Util.Http;
+using Microsoft.AspNetCore.Http;
+
+namespace Datadog.Trace.AppSec.Transports;
+
+internal partial class SecurityTransport
```

Comment:
Mostly a nit, but I don't really understand the "Transport" name here 🤔 This doesn't seems to be anything to do with transport? it's not sending (or receiving) anything?

Reply:
transport was more related to the fact it encapsalutes `HttpContext` stuff but maybe it's not an ideal name indeed
maybe `SecurityManager` ..?

Reply:
Agreed, I think we could find a better name, it seems coordinate all the actions relating to security so maybe SecurityCoordinator or SecurityWorkflow maybe?

Reply:
fixed in 96490b5f2ec03c8bc540ac552bab7d3bce74bdef , to `SecurityCoordinator`